pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -380,6 +380,13 @@ func (r *ReconcileVirtualMachineImport) importDisks(provider provider.Provider,
 				if err = r.endImportAsFailed(provider, instance, foundDv, "dv is in Failed Phase"); err != nil {
 					return err
 				}
+			} else if foundDv.Status.Phase == cdiv1.Pending {
+				instanceNamespacedName := types.NamespacedName{Name: instance.Name, Namespace: instance.Namespace}
+				message := fmt.Sprintf("DataVolume %s is pending to bound", foundDv.Name)
+				processingCond := conditions.NewProcessingCondition(string(v2vv1.Pending), message, corev1.ConditionTrue)
```

Comment:
Do we want to remove this condition once the DV got bound?

Reply:
Right, thanks, it may be there for long time. Do we want to create another condition like 'Bound', or switching condition back to 'CopyingDisks' cond is enough?

Reply:
IMHO, 'CopyingDisks' is better.